.github/workflows/kb_sdk-actions.yml

```diff
@@ -111,19 +125,9 @@ jobs:
           repository: kbaseapps/${{ matrix.app }}
           path: kbase_app
 
-      - name: prep for repo tests
-        run: |
```

Comment:
I guessing you think that running both env and file tests for the app tests is overkill?

Reply:
I can add it in. IMO it's somewhat redundant with the other set of tests that generate new apps, which exercise both env var and test.cfg testing.

Reply:
If you don't think it's necessary I'm fine with that; I just want to understand the reason for the change.

Reply:
I suppose another alternative is running env tests with one app and file tests with the other, but I don't feel strongly about it

Reply:
I'll just switch to the same paradigm as the other tests, checking both the env var and the file tests.